File: src/main/java/net/starlegacy/feature/gas/collectionfactors/HyperspaceOnlyFactor.java
package net.starlegacy.feature.gas.collectionfactors;

import org.bukkit.Location;

public class HyperspaceOnlyFactor extends CollectionFactor {

    @Override
    public boolean factor(Location location) {
        return location.getWorld().getName().contains("Hyperspace");
    }
}

File: server/src/main/kotlin/net/horizonsend/ion/server/features/gas/collectionfactors/HyperspaceOnlyFactor.kt
package net.horizonsend.ion.server.features.gas.collectionfactors

import org.bukkit.Location

class HyperspaceOnlyFactor : CollectionFactor() {
    override fun factor(location: Location): Boolean {
        return location.world.name.contains("Hyperspace", ignoreCase = true)
    }
}
